packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -229,38 +257,42 @@ function executeDispatch(
   event.currentTarget = null;
 }
 
-function executeDispatchesInOrder(
+function processDispatchQueueItemsInOrder(
   event: ReactSyntheticEvent,
-  capture: DispatchQueueItemPhase,
-  bubble: DispatchQueueItemPhase,
+  phase: DispatchQueueItemPhase,
```

Comment:
I find the `phase` naming confusing both in this argument and in the property. When I read `phase` I think of `'capture' | 'bubble'`. But it is actually more like an array of handlers. Can we rename this to `listeners`?

Same with the type name, I thought it refers to the string constants.